packages/block-library/src/classic/edit.js

```diff
@@ -70,13 +70,13 @@ export default class ClassicEdit extends Component {
 		} = this.props;
 
 		const editor = window.tinymce.get( `editor-${ clientId }` );
-		const currentContent = editor.getContent();
+		const currentContent = editor?.getContent();
```

Comment:
The cause of this issue described was fixed by #25162, however we've observed that `null` is a possible return value of `window.tinymce.get`.

Should we proceed with this or shall I close? @ntsekouras